src/main/webapp/scripts/tests/comment-class-test.js

```diff
@@ -0,0 +1,34 @@
+QUnit.test('Comment Class - Constructor (normal use)', function(assert) {
```

Comment:
In this file you should be testing that when you create a comment, it handles the error cases you made and you should also be testing that it gets a toxicity number after mocking the API.